dotcom-rendering/src/components/NumberedTitleBlockComponent.tsx

```diff
@@ -27,26 +24,20 @@ const titleStyles = css`
 	}
 `;
 
-const numberStyles = (palette: Palette) => css`
+const numberStyles = () => css`
```

Comment:
One for us to follow-up on: if this no longer takes parameters it can become a constant rather than a function!